include/jemalloc/internal/arena_inlines_b.h

```diff
@@ -203,13 +204,32 @@ arena_vsalloc(tsdn_t *tsdn, const void *ptr) {
 	return sz_index2size(full_alloc_ctx.szind);
 }
 
+JEMALLOC_ALWAYS_INLINE void
+large_dalloc_safety_checks(edata_t *edata, szind_t szind) {
+	/*
```

Comment:
Guard with a `config_opt_safety_checks` check? We've at least thought about trying to turn the boat on code size, and we otherwise strip the safety check stuff during linking.

Reply:
Makes sense. Added.